.circleci/config.yml

```diff
@@ -115,8 +115,6 @@ jobs:
       - run: mkdir test-results
       - docker-run:
           cmd: go mod download
-      - docker-run:
-          cmd: make check-all-generated-files-up-to-date
```

Comment:
how come?

Reply:
nevermind, just saw https://github.com/weaveworks/eksctl/pull/3113

Reply:
The reason for this PR was that `check-all...` was also being run in the `test` target